src/main/resources/database.properties

```diff
@@ -0,0 +1,6 @@
+driver=postgresql
+host=localhost
+database=my_test_db
```

Comment:
It's better to rename to "directory" (to match the deployment) so we don't have to setup 2 versions for local deployment and docker-compose deployment

Reply:
Renamed database to "directory".